Readme.md

```diff
@@ -20,64 +20,72 @@ The extension is installed by default on the
 [`timescaledev/promscale-extension:latest-pg12`](https://hub.docker.com/r/timescaledev/promscale-extension) docker image.
 
 For bare-metal installations, the full instructions for setting up PostgreSQL, TimescaleDB, and the Promscale Extension are:
+
+1) Install some necessary dependencies
+    ```bash
+    sudo apt-get install -y wget gnupg2 lsb-release
+    ```
 1) [Add the PostgreSQL APT repository (Ubuntu)](https://www.postgresql.org/download/linux/ubuntu/)
-```bash
-echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -c -s)-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
-wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
-sudo apt-get update
-```
-1) Install PostgreSQL 13 and TimescaleDB
-```bash
-sudo add-apt-repository ppa:timescale/timescaledb-ppa
-sudo apt-get update
-sudo apt-get install timescaledb-2-postgresql-13
-```
+    ```bash
+    echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -c -s)-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
+    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+    ```
+1) Add the TimescaleDB APT repository
+    ```bash
+    echo "deb [signed-by=/usr/share/keyrings/timescale.keyring] https://packagecloud.io/timescale/timescaledb/ubuntu/ $(lsb_release -c -s) main" | sudo tee /etc/apt/sources.list.d/timescaledb.list
+    wget --quiet -O - https://packagecloud.io/timescale/timescaledb/gpgkey | sudo apt-key add -
+    ```
+1) Install PostgreSQL with TimescaleDB
+    ```bash
+    sudo apt-get update
+    sudo apt-get install -y timescaledb-2-postgresql-14
+    ```
 1) Tune the PostgreSQL installation
-```bash
-sudo timescaledb-tune --quiet --yes
-sudo service postgresql restart
-```
+    ```bash
+    sudo timescaledb-tune --quiet --yes
+    sudo service postgresql restart
+    ```
 1) Install dependencies for the PGX framework and promscale_extension
-```bash
-sudo apt-get install build-essential clang libssl-dev pkg-config libreadline-dev zlib1g-dev postgresql-server-dev-13
-```
+    ```bash
+    sudo apt-get install -y build-essential clang libssl-dev pkg-config libreadline-dev zlib1g-dev postgresql-server-dev-14
+    ```
 1) [Install rust](https://www.rust-lang.org/tools/install).
-```bash
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
-source $HOME/.cargo/env
-```
-1) Install our fork of the PGX framework
-```bash
-cargo install --git https://github.com/JLockerman/pgx.git --branch timescale cargo-pgx
-```
-1) Initialize the PGX framework using the PostgreSQL 13 installation
-```bash
-cargo pgx init --pg13=/usr/lib/postgresql/13/bin/pg_config
-```
+    ```bash
+    wget --quiet -O - https://sh.rustup.rs | sh
```

Comment:
I think this is insecure compared to the curl. If using wget I /think/ you need to add `--https-only --secure-protocol=TLSv1_2` but maybe it's better to provide the curl rust does?

Reply:
Indeed. I switched to using `wget` throughout to prevent having an install for both `wget` and `curl` at the top of the instruction list. At the end of the day it doesn't really matter though, so I've reverted the `curl` -> `wget` changes.